pageserver/src/page_service.rs

```diff
@@ -184,6 +186,7 @@ pub async fn libpq_listener_main(
     pipelining_config: PageServicePipeliningConfig,
     listener_ctx: RequestContext,
     listener_cancel: CancellationToken,
+    tls_config: Option<Arc<rustls::ServerConfig>>,
```

Comment:
Nit: ctx and cancel come last, typically. I personally would have put the argument next to `auth`. (If you do the change here, do it for the entire call graph)